### DISCUSSION TITLE
Welcome to BC4000 Discussions!

### DISCUSSION THREAD

**Discussion by drwjf** [drwjf/BC4000#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ScythianBeastie** [comment]
Do you know whether the North America map is coming back? It went away when the last update went into use.

**Reply by ScythianBeastie** [reply]
To fulfill the requirement in the developer’s message: I’m a fan of world history games and am really enjoying BC 4000!

**Comment by G1tR4kt** [comment]
Hi, i‘ve been enjoying BC4000 for quite some time now, mastered5th level of difficutly and 200x100 earth map. Ypur game is really amazing, but there are few small QOL features, i‘d like to see - in setting, custom city advisor building order with conditions (like always build markets if city produces >10 gold and >6 shield or simply let us change the default order to match game style) and some modes for workers/engineers (for instance, build road between two cities or focus on a certain city). When playing on even middle-sized map, after around 40 cities management becomes straight up anoying and is less and less about „patience“ (which works great on <100x100 maps tho). Perhaps this custom management can be an option only for certain governments (?, to keep balance or realism). Anyway, your game is majestic and i wish you to keep up the awesome work. I have some other ideas too, but i understand that developer‘s job is not that easy, so i better keep them for later))
